### PR TITLE
updated suggested searches

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by JainM6** [comment]
@kimsauce Could you please help me to find out the issue with this PR? The error says incorrect path, but I am able to generate the local build.

**Comment by kimsauce** [comment]
> @kimsauce Could you please help me to find out the issue with this PR? The error says incorrect path, but I am able to generate the local build.

**Comment by kimsauce** [comment]
> Could you please help me to find out the issue with this PR? The error says incorrect path, but I am able to generate the local build.

@JainM6 it's saying this link doesn't exist. It's prob in the CID file. In Atom, you can do a **Find All** to locate it and then fix it.  
Screenshot 2023-02-01 at 9 59 45 AM

**Comment by JainM6** [comment]
> 

@kimsauce: I figured out the incomplete paths, and fixed it now. It's sorted.

File: docs/search/get-started-with-search/suggested-searches/index.md [hunk 1]
```diff
@@ -6,11 +6,40 @@ description: Sumo Logic queries can help track and diagnose common IT issues. Ta
...
 ## Guide contents
 
 In this section, we'll introduce the following concepts:
 
-<DocCardList items={useCurrentSidebarCategory().items}/>
+<div className="box-wrapper" markdown="1">
+<div className="box box1 card">
+  <div className="container">
+  <h4><a href="/docs/search/get-started-with-search/suggested-searches/apache-access-parser">Apache Access Parser</a></h4>
+  <p>The following searches were built for use with the Apache Access Parser.</p>
```
Comment (kimsauce) [inline docs/search/get-started-with-search/suggested-searches/index.md:18]: These descriptions need to make sense as standalone statements on a separate page. You could say something like "Learn about suggested searches for the Apache Access Parser".

Comment (JainM6) [reply docs/search/get-started-with-search/suggested-searches/index.md:18]: I did not write it on my own. It has been there in the system for a long I guess. But yes, we should change them.

Comment (kimsauce) [reply docs/search/get-started-with-search/suggested-searches/index.md:18]: Is this tied to an urgent feature release? Apologies if I misunderstood; I thought this was my Hub Pages project.

To clarify, as we're building out our assigned Hub Pages (I thought that's what this PR was) I didn't mean to pull descriptions from the `description` field of that particular doc. I've written 90% of them from scratch.

Comment (JainM6) [reply docs/search/get-started-with-search/suggested-searches/index.md]: @kimsauce Thank you so much for your contribution, you're doing awesome. For this doc, we should be good now as I have descriptions.

**Review by kimsauce — APPROVED** [review]
Looks great!

**Pull request merged**